index.html

```diff
@@ -0,0 +1,201 @@
+          <li class="nav-item"><a class="nav-link active" href="#program">Program</a></li>
+          <li class="nav-item"><a class="nav-link active" href="#speakers">Speakers</a></li>
+          <li class="nav-item"><a class="nav-link active" href="#partners">Sponsors</a></li>
+          <li class="nav-item"><a class="nav-link active" href="#">CB Campain</a></li>
+        </ul>
+      </div>
+        
+      <!------End popup------------------>
+
+      </header>
+      <main>
+        <section id="landing-p">
+          <div class="container">
+            <div class="row">
+              <div class="col-1"></div>
+              <div class="col-10 mt-5 mb-2 pt-5">
+                <h1>"Hello! ICT World"</h1>
+                <h2 class="mb-5">The east african software<br>developers' Community Annual symposium<br>2022</h2>
+                <p id="intro">
+                  I am excited to inform you about our annual symposium .This symposium brings together the 
+                  community of experts,academics and activists who comprise the Standard code Academy, affiliate 
+                  network in different countries every year.This year we we hope to expand our invitation list including organisations and individuals.
+                </p>
+                <h3 class="mt-5">2022.12.22(THU) <span>&#126;</span> 2022.12.30(FRI)</h3>
+                <p id="venue">&#64; Serena Hotel, Kampala Uganda</p>
+              </div>
+              <div class="col-1"></div>
+            </div>
+          </div>
+        </section>
+        <section class="program" id="program">
+          <div class="row program-title container-fluid mx-auto">
+            <div class="col"></div>
+            <div class="col col- mt-3 pt-3"><h3>Main Program</h3></div>
+            <div class="col"></div>
+          </div>
+          <div class="row program-title container mx-auto">
+            <div class="col"></div>
+            <div class="col"><hr id="line"></div>
+            <div class="col"></div>
+          </div>
+          <div class="container-fluid">
+            <div class="row mt-3 cards mx-auto programs">
+              <div class=" col-md-2 card">
+                <img src="./images/speak.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Lecture</h5>
+                  <p class="card-text">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                  </p>
+                </div>
+              </div>
+              <div class="col-md-2 card">
+                <img src="./images/exhibit.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Exhibition</h5>
+                  <p class="card-text">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                  </p>
+                </div>
```

Comment:
- [ ] kindly leave some space between the cards
`your desing`
![image](https://user-images.githubusercontent.com/66230306/192095660-ddfc9480-e018-4425-841e-cc4677a44c70.png)

`figma design`
![image](https://user-images.githubusercontent.com/66230306/192095690-74de3f0d-c6cf-4e96-a381-b2f1ed7b3263.png)